clang/include/clang/Options/Options.td

```diff
@@ -5172,6 +5166,14 @@ def ffuchsia_api_level_EQ : Joined<["-"], "ffuchsia-api-level=">,
   Group<m_Group>, Visibility<[ClangOption, CC1Option]>,
   HelpText<"Set Fuchsia API level">,
   MarshallingInfoInt<LangOpts<"FuchsiaAPILevel">>;
+def fcfg_block_num_threshold
```

Comment:
Suggestion: fexperimental-lifetime-safety-max-num-cfg-blocks

Also move it next to the other lifetime safety options